### PR TITLE
FR - Adding missing translations in Methodology (fix #626)

### LINKED ISSUE #626: Methodology French Translation - extra changes needed
@borisschapira we added a this line to line [386](https://github.com/housseindjirdeh/almanac.httparchive.org/blob/b68deaa02581b8b59dc422d396de928878030c74/src/templates/en/2019/methodology.html#L386) of the Methodology section in #592:

        All detection tools, including Wappalyzer, have their limitations. The validity of their results will always depend on how accurate their detection mechansims are. The Web Almanac will add a note in every chapter where Wappalyzer is used but its analysis may not be accurate due to a specific reason. 

Any chance you could update the French Translation to include this too?

CC: @housseindjirdeh 

And also the warning on BigQueries from https://github.com/HTTPArchive/almanac.httparchive.org/pull/618

### PR DESCRIPTION
**Pull request by borisschapira**
Fixes #626

### CODE REVIEW

File: src/templates/fr/2019/methodology.html [hunk 1]
```diff
@@ -163,6 +163,10 @@ <h2 id="dataset">À propos du jeu de données</h2>
 
         <p>
         Par exemple, pour obtenir le nombre médian d’octets de JavaScript servis sur les pages pour ordinateurs de bureau et périphériques mobiles, regardez <a href="https://github.com/HTTPArchive/almanac.httparchive.org/blob/master/sql/2019/01_JavaScript/01_01b.sql">01_01b.sql</a>:
+        
+
+        <p>
+          Par exemple, pour comprendre le nombre médian d'octets de JavaScript sur les sites pour ordinateurs de bureau et périphériques mobiles, voir <a href="https://github.com/HTTPArchive/almanac.httparchive.org/blob/master/sql/2019/01_JavaScript/01_01b.sql">01_01b.sql</a>:
```
Comment (bazzadp) [inline src/templates/fr/2019/methodology.html]: You seem to have repeated the previous paragraph?

This was the line to translate above that paragraph:

```
        <p>
          <aside>Please note that some of these queries are quite large and can be <a href="https://cloud.google.com/bigquery/pricing">expensive</a> to run yourself, as BigQuery is billed by the terabyte. For help controlling your spending, refer to Tim Kadlec's post <a href="https://timkadlec.com/remembers/2019-12-10-using-bigquery-without-breaking-the-bank/">Using BigQuery Without Breaking the Bank</a>.</aside>
        </p>
```

Comment (bazzadp) [reply src/templates/fr/2019/methodology.html]: Though nice to know your translation is semi-consistent 😊

Though a couple of words different so pick which one you like and change in this same PR if you want.

**Comment by borisschapira** [comment]
Sorry @bazzadp, I was careless and I got fooled by the addition of another paragraph (on Big Query costs). I removed the duplicated one and translated this other paragraph.

**Comment by bazzadp** [comment]
No probs!

Thanks for doing that. As it's only a small additional translation I'll go ahead and merge rather than await a formal review from another French speaker.

**Pull request merged**